cmd/nitro/nitro.go

```diff
@@ -350,7 +351,8 @@ func main() {
 	defer cancelFunc()
 
 	vcsRevision, vcsTime := genericconf.GetVersion()
-	nodeConfig, l1Wallet, l2DevWallet, l1Client, l1ChainId, err := ParseNode(ctx, os.Args[1:])
+	args := os.Args[1:]
+	nodeConfig, l1Wallet, l2DevWallet, l1Client, l1ChainId, err := ParseNode(ctx, args)
```

Comment:
(?) instead of using ParseNode directly, it could be done by LiveNodeConfig.NewLiveNodeConfig, but that would probably require:
 *  storing in LiveNodeConfig or returning from NewLiveNodeConfig following vars:  l1Wallet, l2DevWallet, l1Client, l1ChainId
 * moving to ParseNode all the below logic which sets some fields of NodeConfig based on some other fields (it probably should be done either way to keep those derived fields in sync after config reload)